src/gui/screen_menu_lan_settings.cpp

```diff
@@ -23,7 +23,14 @@ ScreenMenuConnectionBase::ScreenMenuConnectionBase(uint32_t dev_id, const char *
 
 void ScreenMenuConnectionBase::refresh_addresses() {
     char str[ADDR_LEN];
-
+    char fwver[4];
```

Comment:
`fwver` should be defined inside the if then branch.

Reply:
Done.